### ISSUE TITLE
What to improve further?

### ISSUE THREAD

**Issue by xHansonx** [chen0040/keras-text-summarization#6]
Based on the results you showed in gits, it looks like losses from training and valid is diverging instead of converging as number of epoch increasing. So, what's the problem here?  I saw other text stigmatization model, they build their own dictionary and input something like one-hot encoding vector for each word. Does this process make difference? Thank you! 
I'm glad if you want to talk more by WeChat (ID: XHansonX)

**Comment by rmahfuz** [comment]
I agree that it is strange that the validation loss increases instead of decreasing. Does anyone have suggestions about how this can be fixed? Thanks!
